### PR TITLE
Add css to the /windows and /apple templates

### PR DESCRIPTION
**Pull request by atorregrosa-smd**
I added a bit of css to make the configuration guides look better, as they were very rough.

- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

### CODE REVIEW

**Comment by kradalby** [comment]
Thanks, would you mind posting a screenshot?

**Comment by atorregrosa-smd** [comment]
imagen

File: templates/apple.html [hunk 1]
```diff
@@ -4,11 +4,25 @@
...
+      }
+      h1,h2,h3 {
+        line-height: 1.2;
+      }
+    </style>
   </head>
 
   <body>
-    <h1>headscale</h1>
+    <h1>headscale Apple configuration </h1>
```
Comment (kradalby) [inline templates/apple.html:25]:
```suggestion
    <h1>headscale: Apple configuration </h1>
```

File: templates/windows.html [hunk 2]
```diff
@@ -4,11 +4,25 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Windows - headscale</title>
```
Comment (kradalby) [inline templates/windows.html:7]:
```suggestion
    <title>headscale - Windows</title>
```

**Comment by atorregrosa-smd** [comment]
imagen

File: templates/windows.html [hunk 3]
```diff
@@ -4,11 +4,25 @@
...
+      }
+      h1,h2,h3 {
+        line-height: 1.2;
+      }
+    </style>
   </head>
 
   <body>
-    <h1>headscale</h1>
+    <h1>headscale Windows configuration</h1>
```
Comment (kradalby) [inline templates/windows.html:25]:
```suggestion
    <h1>headscale: Windows configuration</h1>
```

File: templates/apple.html [hunk 4]
```diff
@@ -4,11 +4,25 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Apple - headscale</title>
```
Comment (kradalby) [inline templates/apple.html:7]:
```suggestion
    <title>headscale - Apple</title>
```

**Review by kradalby — COMMENTED** [review]
Looks good!, a few nits when we are at it.

**Comment by kradalby** [comment]
Few comments and if you run `prettier` to format them, then we are good.

**Review by kradalby — APPROVED** [review]

**Pull request merged**